config.yml

```diff
@@ -7,4 +7,4 @@ since: '2023-01-01'
 
 # GitHub Pages path to the repository
 # Used for handling relative paths for assets
-basePath: '/sbv-world-health-org-metrics'
+basePath: '/org-metrics-dashboard'
```

Comment:
does this path exist? I don't see it in the repo... Or maybe I'm misunderstanding what this does.

Reply:
This is for the github pages link 😄 

https://github-community-projects.github.io/org-metrics-dashboard/

Notice how the stylesheet is busted 😢